tests/tools.py

```diff
@@ -22,6 +22,7 @@
 VER_MAJOR_MINOR = sys.version_info[:2]
 VER_FULL = sys.version_info[:3]
 ON_MAC = (platform.system() == 'Darwin')
+ON_WINDOWS = (platform.system() == 'Windows')
```

Comment:
shoudl we just import this from `xonsh.platform`?  same for `ON_DARWIN` (instead of `ON_MAC`)